metadata-ingestion/tests/integration/remote/golden/remote_enricher_golden.json

```diff
@@ -158,5 +158,35 @@
         "lastObserved": 1629795600000,
         "runId": "remote-1"
     }
+},
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:hive,SampleHiveDataset,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "status",
+    "aspect": {
+        "json": {
+            "removed": false
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1629795600000,
+        "runId": "remote-1"
+    }
+},
+{
+    "entityType": "tag",
+    "entityUrn": "urn:li:tag:Legacy",
+    "changeType": "UPSERT",
+    "aspectName": "tagKey",
```

Comment:
Getting a tag key aspect, just want to make sure this makes sense for this source